test/unit/L2/l2LPTGateway.test.ts

```diff
@@ -68,11 +74,36 @@ describe('L2 Gateway', function() {
       to: await mockL1GatewayL2Alias.getAddress(),
       value: ethers.utils.parseUnits('1', 'ether'),
     });
+
+    arbSysMock = await smock.fake(ArbSysABI.abi, {
```

Comment:
Mock with interface instead of checking in ABI?

Reply:
resolved in #18